src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -101,6 +104,18 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
             }
         }
 
+        private async Task DocumentChangedAsync(Document oldDocument, Document newDocument)
+        {
+            var oldTextVersion = await oldDocument.GetTextVersionAsync(CancellationToken.None).ConfigureAwait(false);
+            var newTextVersion = await newDocument.GetTextVersionAsync(CancellationToken.None).ConfigureAwait(false);
+
+            if (oldTextVersion != newTextVersion)
+            {
+                UpdateClassVersion();
```

Comment:
Well, given there's two versions (class and members), I'm guessing there's some other code somewhere else updating one but not the other. Where does that run?